### PR TITLE
Hello microverse project

### PR DESCRIPTION
**Pull request by MohammedElseddik**
## Hello Microverse first project

In this project I:

 - Set up a linter for HTML and CSS to the web page.
 - Created a simple web page and committed and pushed changes to the Github repo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,13 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Hello-world
```
Comment (ShoiraTa) [inline README.md:3]: - [ ] Please add more sections in Readme file, like project description, Author, Acknowledgements. You can use [this template](https://github.com/microverseinc/readme-template) and leave only sections that you need for this project.

File: README.md [hunk 2]
```diff
@@ -1,15 +1,13 @@
...
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- Major languages: HTML, CSS
 
 ## Live Demo (if available)
```
Comment (ShoiraTa) [inline README.md:13]: - [ ] I can see that live demo section in `readme` is empty, if so I suggest you to delete this section, in the future when you start deploying projects you can insert link here 🤗

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,3 @@
+app_screenshot.png
+MIT.md
+test.md
```
Comment (ShoiraTa) [inline .gitignore:3]: - [ ] OPTIONAL: I suggest you to always add node_modules into `.gitignore` file, node_modules package is usually a folder with all the dependencies that you will install for your project, and generally its massive with lots of data and files. If you do not add it into `.gitignore` file, all those files will be committed to your github repository which is totally unnecessary.

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
Hi @MohammedElseddik ,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

**First let me to point out:**

- Great commit history! ✔️ 
- You passed all the Linters! ✔️ 
- Professional pull request! ✔️ 

Keep on coding, good job so far! :clap:

There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

_Check the comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 4]
```diff
@@ -1,72 +1,24 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Hello-world
```
Comment (elmar8287) [inline README.md:3]: - [ ] Nice job, you have added the Authors section, but you have still made your README file more informative, as requested by the previous reviewer. Try to add also the Acknowledgements section 👍

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi @MohammedElseddik , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ]  _Check the comments under the review._

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 5]
```diff
@@ -2,23 +2,39 @@
...
 
-
 ## Authors
 
-👤 **Rebbouh MOhammed ELseddik**
+👤 **Mohammed Elseddik Rebbouh**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
```
Comment (keroloussamy) [inline README.md:20]: [OPTIONAL] Kindly change these links to your links, so anyone can access you.

**Review by keroloussamy — APPROVED** [review]
Hi @MohammedElseddik , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

clap


## Optional suggestions
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**